lib/rubygems/specification.rb

```diff
@@ -1349,16 +1348,15 @@ def self._load(str)
     spec.instance_variable_set :@required_rubygems_version, array[7]
     spec.instance_variable_set :@original_platform,         array[8]
     spec.instance_variable_set :@dependencies,              array[9]
-    spec.instance_variable_set :@rubyforge_project,         array[10]
```

Comment:
I think this change would require bumping the spec version -- it might be better to keep the offsets the same, and just never use the rubyforge ivar?

Reply:
:+1: for keeping the offsets the same. You should probably also add a note explaining that it's deprecated/no longer used.